integrations/torchvision/README.md

```diff
@@ -0,0 +1,202 @@
+<!--
+Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# SparseML Torchvision Integration
+
+By integrating with Torchvision, SparseML enables you to train inference-optimized sparse versions of popular image classification models like ResNet-50 on your dataset.
+
+There are two pathways:
+- **Sparse Transfer Learning**: fine-tune a pre-sparsified checkpoint on your own dataset **[RECOMMENDED]**
+- **Sparsification from Scratch**: Apply state-of-the-art training-aware pruning and quantization algorithms to arbitrary Torchvision models.
+
+Once trained, SparseML enables you to export models to the ONNX format, such that they can be deployed with DeepSparse for GPU-class performance on the CPU.
+
+## Installation
+
+Install with `pip`:
+
+```bash
+pip install sparseml[torchvision]
+```
+
+## Tutorials
+
+- [Sparse Transfer Learning with the CLI](tutorials/sparse-transfer-learning.md)
+- [Sparse Transfer Learning with the Python API](tutorials/docs-torchvision-python-transfer-imagenette.ipynb)
+- Sparsification from Scratch with the CLI (coming soon!)
+- Sparsification from Scratch with the Python API (coming soon!)
+
+## Quick Tour
+
+### SparseZoo
+
+Neural Magic has pre-sparsified versions of common Torchvision models such as ResNet-50. These models can be deployed directly or can be fine-tuned onto custom dataset via sparse transfer learning. This makes it easy to create a sparse image classification model trained on your dataset.
```

Comment:
can be fine-tuned onto custom dataset -> can be fine-tuned onto a custom dataset